### ISSUE TITLE
Hemingway style

### ISSUE THREAD

**Issue by moltar** [errata-ai/packages#17]
Would it be possible to have something like [Hemingway Editor](https://hemingwayapp.com/) functionality as a style?

**Comment by jdkato** [comment]
Yes, it would be possible to have something similar at least. I'd say the [write-good](https://vale.sh/hub/write-good/) package and the [`metric`](https://vale.sh/docs/topics/styles/#metric) extension point would be good starting points.
